### PR TITLE
Measure view-graph estimation runtime

### PR DESCRIPTION
**Pull request by johnwlambert**

### CODE REVIEW

File: gtsfm/view_graph_estimator/cycle_consistent_rotation_estimator.py [hunk 1]
```diff
@@ -138,7 +140,14 @@ def run(
...
 
-        logger.info("Found %d consistent rel. rotations from %d original edges.", len(valid_edges), len(input_edges))
+        end_time = time.time()
+        duration_sec = end_time - start_time
+        logger.info(
+            "Found %d consistent rel. rotations from %d original edges in %.2f sec.",
+            len(valid_edges),
+            len(input_edges),
+            duration_sec,
+        )
```
Comment (akshay-krishnan) [inline gtsfm/view_graph_estimator/cycle_consistent_rotation_estimator.py:150]: could you add a metric? this will drown in all the noise.

Comment (johnwlambert) [reply gtsfm/view_graph_estimator/cycle_consistent_rotation_estimator.py:150]: I think we may not need one, but we could add in a follow-up PR. Logged stats indicate this part is <1 sec for all runs.

Comment (johnwlambert) [reply gtsfm/view_graph_estimator/cycle_consistent_rotation_estimator.py:150]: (I believe the logged stat info is sufficient for now)

Comment (akshay-krishnan) [reply gtsfm/view_graph_estimator/cycle_consistent_rotation_estimator.py:150]: interesting, thanks!

**Review by akshay-krishnan — APPROVED** [review]

**Pull request merged**